frontend/packages/dev-console/src/components/project-access/ProjectAccessForm.tsx

```diff
@@ -16,39 +20,59 @@ const ProjectAccessForm: React.FC<ProjectAccessFormProps> = ({
   errors,
   dirty,
   roles,
+  roleBindings,
+  values,
 }) => {
   const { t } = useTranslation();
+  const [isStaleInfo, setIsStaleInfo] = React.useState<boolean>(false);
   const disableSubmit = !dirty || !_.isEmpty(errors);
+
+  React.useEffect(() => {
+    if (!_.isEqual(roleBindings, values)) {
+      setIsStaleInfo(true);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [roleBindings]);
+
+  const onReload = () => {
+    handleReset();
+    setIsStaleInfo(false);
+  };
```

Comment:
```suggestion
// eslint-disable-next-line react-hooks exhaustive-deps
const isStaleInfo = React.useMemo(() => !_.isEqual(roleBindings, values), [roleBindings]);
```

Reply:
With this change, if the list is updated then on reload the stale info alert does not get removed. This happens because on reload the `values` is updated and the useMemo here is triggered only when `roleBindings` is updated